ANSWERS.md

```diff
@@ -0,0 +1,12 @@
+1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+    A. Describe() breaks the test suite into components making it easier to read.  It() is used inside describe() and 
```

Comment:
Yup 👍 

Also a good thing about describe is it groups the all tests that are dealing with a single aspect. Like a route or function